frontend/src/i18n/locales/en-US.json

```
@@ -295,6 +308,7 @@
     "address-copied": "Address copied to clipboard",
     "connected-with": "Connected with",
     "copy-address": "Copy address",
+    "send-history": "Send history",
```

```suggestion
    "view-send-history": "View send history",
```

cc @garyghayrat for translation

---

查看发送历史

---

Fix here https://github.com/ScopeLift/umbra-protocol/pull/465/commits/9121397c9a0f7060b5265addbdd9fcb5d0cd9f41